app/assets/javascripts/discourse/controllers/composer.js.es6

```diff
@@ -92,6 +111,25 @@ export default Ember.Controller.extend({
     return currentUser && currentUser.get('staff') && this.siteSettings.enable_whispers && action === Composer.REPLY;
   },
 
+  @computed("model.composeState")
+  popupMenuOptions(composeState) {
+    const self = this;
+
+    if (composeState === 'open') {
+      return _popupMenuOptionsCallbacks.map(callback => {
+        let option = callback();
+
+        if (option.condition) {
+          option.condition = self.get(option.condition);
+        } else {
+          option.condition = true;
+        }
+
+        return option;
+      });
+    }
+  },
```

Comment:
Sure that seems better. In this case you don't need to use `self` though because `this` is set by the fat arrow `=>` syntax